packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -18,7 +18,7 @@
           #default="nextQueryScope"
           :suggestion="baseScope.suggestion"
           :highlightCurated="highlightCurated"
-          class="x-next-queries__suggestion"
+          class="x-next-queries__suggestion x-suggestion"
```

Comment:
Hi! I think this `x-next-queries__suggestion` class can be removed

Reply:
I would leave them, removing them is a breaking change, we should decide what to do with BEM like classes in the project. This topic should be brought up in some meeting (refinement or standup maybe)

Reply:
Yep, I would avoid removing those classes for now to prevent breaking changes